### PR TITLE
Use parse component instead of parse5

### PR DESCRIPTION
**Pull request by jackmellis**
@qt911025

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,25 +47,7 @@ module.exports = function ({ content, filename, sourceMap }) {
       + transpile(`${exportsTarget}.staticRenderFns = [${staticRenderFns}];`)
       + `\n${exportsTarget}.render._withStripped = true;`;
   }
-  const result = `${scriptPart}${compiledTemplate}`;
```
Comment (qt911025) [inline index.js]: Why are source maps removed?

PS: Sry. My computer is broken, which costs me 2 days to recover data.

Comment (jackmellis) [inline index.js]: The parseComponent function adds padding where the template/style tags were, so the script is still on the same line as the original vue file. This means you don't need source maps, which cuts down on overhead a little!

Comment (qt911025) [inline index.js]: You mean that parseComponent will add lines to fill original `styles` and `template` part?

If there is no problem while consuming break point debugging, just merge.

Comment (jackmellis) [inline index.js]: Yes it should be fine (in theory). So a component like:
```html
<template>
<div><slot></slot></div>
</template>
<style>
div{}
</style>
<script>
export default {}
</script>
```
will come out like
```html
//
//
//
//
//
//
//
exports.default = {}
exports.default.render = ...
```
something like that. So debugging should be kept intact. I might debug a few components like this to check it's right and then I'll merge it

Comment (jackmellis) [inline index.js]: Just debugged a few components using this method and it works probably even better than with source maps.

**Pull request merged**